.coveragerc

```diff
@@ -0,0 +1,5 @@
+# .coveragerc to control coverage.py
+[run]
+
+[report]
+include = */site-packages/google/datalab/*
```

Comment:
What if you include `*/google/datalab/*` and exclude `site-packages`? Does it figure out how to map the coverage info to the source files?